lib/engine/config/game/g_1835.rb

```diff
@@ -0,0 +1,583 @@
+# frozen_string_literal: true
+
+# TODO: figure out the status of the following in this config.
+# trains are correct
+# cert limits are correct
+# cash is correct
+# phases aren't checked yet.
+# tiles are not correct.
+# market is not correct
+# map hexes are not correct
+# companies are not correct.
+# minors aren't there
+# privates aren't correct.
+
+# File original exported from 18xx-maker/export-rb
+# https://github.com/18xx-maker/export-rb
+# rubocop:disable Lint/RedundantCopDisableDirective, Layout/LineLength, Layout/HeredocIndentation
+
+module Engine
+  module Config
+    module Game
+      module G1835
+        JSON = <<-'DATA'
+{
+  "filename": "1835",
+  "modulename": "1835",
+  "currencyFormatStr": "%dM",
+  "bankCash": 12000,
+  "certLimit": {
+    "3": 19,
+    "4": 15,
+    "5": 12,
+    "6": 11,
+    "7": 9
+  },
+  "startingCash": {
+    "3": 600,
+    "4": 475,
+    "5": 390,
+    "6": 340,
+    "7": 310
+  },
+  "capitalization": "full",
+  "layout": "pointy",
+  "mustSellInBlocks": false,
+  "locationNames": {
+    "A11": "Kiel"
+  },
+  "tiles": {
+    "3": 2,
+    "5": 2,
+    "6": 2,
+    "7": 2,
+    "8": 5,
+    "9": 5
+  },
+  "market": [
+    [
+      "75",
+      "80",
+      "90",
+      "100p",
+      "110"
+    ],
+    [
+      "70",
+      "75",
+      "80",
+      "90p",
+      "100"
+    ]
+  ],
+  "companies": [
+    {
+      "name": "Private with an icon",
+      "sym": "P1",
+      "value": 100,
+      "revenue": 5,
+      "desc": "Description"
+    },
+    {
+      "name": "Private with a company",
+      "sym": "P1",
+      "value": 140,
+      "revenue": 10,
+      "desc": "Description"
+    },
+    {
+      "name": "Private with a token",
+      "sym": "P1",
+      "value": 160,
+      "revenue": 15,
+      "desc": "Description",
+      "min_players": 3
+    },
+    {
+      "name": "Private with a tile",
+      "sym": "P1",
+      "value": 160,
+      "revenue": 15,
+      "desc": "This tile (2) is aliased in this game to 57"
+    },
+    {
+      "name": "Private with a custom tile",
+      "sym": "P1",
+      "value": 180,
+      "revenue": 20,
+      "desc": "This tile is defined in the game file"
+    },
+    {
+      "name": "Private with Hex and Note",
+      "sym": "P1",
+      "value": 220,
+      "revenue": 30,
+      "desc": "Description"
+    }
+  ],
+  "corporations": [
+    {
+      "sym": "MS",
+      "name": "Black Railroad",
+      "logo": "1835/MS",
+      "tokens": [
+        0,
+        40,
+        100
+      ],
+      "coordinates": "C11",
+      "color": "red"
```

Comment:
Shouldn't it be black? ;-)

Reply:
placeholder only. THey're all getting cleaned up in the next commit, ~none of the company stuff was correct from where I started, but at least this renders a map and keeps each commit phase relatively small.